Assets/Scripts/Lobby/LobbyManager.cs

```diff
@@ -42,9 +43,17 @@ private void GetLobby()
             var lobby = task.GetAwaiter().GetResult();
             if (lobby != CurrentLobby)
             {
+                var playerIds = new string[4];
```

Comment:
Can we use lobby.Players.Count instead, since we aren't sure if there are actually 4 players in the lobby? Not an issue here but we risk handing over a list with empty spots to the play button.